src/js/stores/MarathonStore.js

```diff
@@ -213,6 +218,35 @@ var MarathonStore = Store.createStore({
     // Handle ongoing request here.
   },
 
+  processMarathonServiceVersions({serviceId, versions}) {
+    versions = versions.reduce(function (map, version) {
+      return map.set(version);
+    }, new Map());
+
+    this.emit(MARATHON_SERVICE_VERSIONS_CHANGE, {serviceId, versions});
+  },
+
+  processMarathonServiceVersionsError() {
+    this.emit(MARATHON_SERVICE_VERSIONS_ERROR);
+  },
+
+  processMarathonServiceVersion({serviceId, version, versionId}) {
+    // TODO (orlandohohmeier): Convert version into typed version struct
+    this.emit(MARATHON_SERVICE_VERSION_CHANGE, {serviceId, versionId, version});
+  },
+
+  processMarathonServiceVersionError() {
+    this.emit(MARATHON_SERVICE_VERSION_ERROR);
+  },
+
+  fetchServiceVersion: function (serviceId, versionId) {
+    MarathonActions.fetchServiceVersion(serviceId, versionId);
+  },
+
+  fetchServiceVersions: function (serviceId) {
+    MarathonActions.fetchServiceVersions(serviceId);
+  },
```

Comment:
Normally we tend to put these `#fetch*` methods towards the top after `#removeChangeListener`.
Could we move them, just so that it's consistent?